test/TestUtils/TestProjectFixture.cs

```diff
@@ -70,7 +70,7 @@ public class TestProjectFixture
                 ?? Path.Combine(AppContext.BaseDirectory, s_testArtifactDirectoryEnvironmentVariable);
 
             _sdkDotnet = new DotNetCli(dotnetInstallPath ?? DotNetCli.GetStage0Path(repoDirectoriesProvider.RepoRoot));
-            _currentRid = currentRid ?? _sdkDotnet.GetRuntimeId();
+            _currentRid = repoDirectoriesProvider.TargetRID;
```

Comment:
Agreed - I had missed it in the initial push and had fixed it locally already. will be part of the next push.